keyboards/aplx6/keymaps/Media/keymap.c

```diff
@@ -0,0 +1,38 @@
+#include QMK_KEYBOARD_H
```

Comment:
This keymap should either be renamed to default, or renamed to lower case and a default keymap added